src/main/java/de/tum/cit/aet/artemis/plagiarism/service/PlagiarismPostService.java

```diff
@@ -90,6 +90,11 @@ public Post createPost(Long courseId, Post post) {
         return savedPost;
     }
 
+    public void informInstructorAboutPostReply(long postId) {
+        Post post = postRepository.findPostByIdElseThrow(postId);
```

Comment:
I don't know this code area too much but could we check for `post.getPlagiarismCase() != null` to ensure that it's a plagiarism-related post?
`Post#plagiarismCase` seems to be a EAGER one-to-one relationship which would not require to make an additional DB call AFAICS.